learnStrings/learnString.js

```diff
@@ -0,0 +1,38 @@
+// most repeated character in a string
+export const findMostRepChar = str => {
+    let testObj = {}
+    let mostRepChar;
+    let mostRepCharCount = Number.NEGATIVE_INFINITY;
+    for (let char in str) {
+        testObj[str[char]] ? testObj[str[char]] += 1 : testObj[str[char]] = 1
+    }
+    for (let char in testObj) {
+        if (testObj[char] > mostRepCharCount) {
+            mostRepCharCount = testObj[char];
+            mostRepChar = [char]
+        } else if (testObj[char] = mostRepCharCount) {
+            mostRepChar.push(char)
+        }
+    }
+    return mostRepChar;
+}
+// check weather a string is palindrome or not
+
+export const checkPalindrome = str => {
```

Comment:
rename function name to `isPalindrome`.  And return boolean values.

Reply:
changed